docs/en_US/contributor-guidance.md

```diff
@@ -0,0 +1,53 @@
+
+## Introduction to the Project
+
+The ob-operator is a Kubernetes operator that simplifies the deployment and management of OceanBase database cluster and related resources on Kubernetes.
+
+By contributing to our project, you'll help achieve simplicity of controlling OceanBase resource like clusters, tenants, backups and so on, making its users more easier to manage OceanBase database and combine on various versions or distributions of Kubernetes.
+
+Whether you are fixing bugs, adding new features, or improving documentation, your work will make a significant impact on the project and its users.
+
+## Code of Conduct
+
+Before you start contributing, we strongly encourage you to review our [Code of Conduct](https://github.com/oceanbase/ob-operator?tab=coc-ov-file). It outlines the expectations for behavior within our community and provides a transparent process for reporting unacceptable behavior. We are committed to providing a welcoming and supportive environment for all contributors.
+
+## Getting Started with Development
+
+To get started with developing for our project, please refer to our [Development Guidelines](./develop-locally.md). This document provides the necessary steps to set up the development environment on your laptop.
+
+## Deploying Changes
+
+If you are working on changes that require deployment, look at our [Deployment Guide](./develop-locally.md) for detailed instructions on how to deploy and test your changes in a production-like environment. This ensures that all changes are vetted for stability and compatibility before being integrated.
+
+## How to Contribute
+
+Here is a brief overview of how you can contribute to our project:
+
+1. Familiarize Yourself: Get to know the project by reading the documentation, understanding the codebase, and identifying the areas you are interested in.
+
+2. Find an Issue: Look at our issue tracker to find tasks that interest you. We also welcome contributions that improve the project in ways not yet identified in our tracker.
+
+3. Communicate: Before starting to work on an issue, let the community know to avoid duplicate efforts and to get any necessary input or guidance.
+
+4. Fork and Clone: Fork the repository and clone it to your local machine. This will be your private workspace.
+
+5. Make Changes: Implement your changes in your local environment adhering to our development practices and coding standards.
+
+6. Write Tests: Ensure your changes are tested. We strive to maintain a high level of code quality and reliability.
+
+7. Document Your Work: Update existing documentation or create new documentation as needed. Good documentation is just as important as good code.
+
+8. Create a Pull Request: Once your changes are ready, submit a pull request. Provide a clear description of the problem and solution, including any relevant issue numbers.
+
+9. Code Review: Engage with the code review process, addressing any feedback provided by the project maintainers.
```

Comment:
Code Review is for maintainers, for contributors, this should be "Modify your code"